pkg/collector/corechecks/snmp/internal/report/report_device_metadata.go

```diff
@@ -72,6 +73,14 @@ func (ms *MetricSender) ReportNetworkDeviceMetadata(config *checkconfig.CheckCon
 			interfaceTags = append(interfaceTags, "interface_alias:"+interfaceStatus.Alias)
 		}
 		interfaceTags = append(interfaceTags, tags...)
+
+		// append user's custom interface tags
+		interfaceCfg, err := getInterfaceConfig(ms.interfaceConfigs, interfaceIndex, interfaceTags)
+		if err != nil {
+			log.Errorf("failed to tag interface status metric with user tags: %s", err.Error())
```

Comment:
```suggestion
			log.Tracef("failed to tag interface status metric with user tags: %s", err.Error())
```

Should be trace log instead? This seems something that might happen often ("no matching interface found" error).

Reply:
Yes, nice catch.